### PR TITLE
Feature/dtrunetwo 65 v4 remove filters

### PR DESCRIPTION
**Pull request by Rrathinasabapath**
Please review and approve.

### CODE REVIEW

File: src/Hyperwallet.js [hunk 1]
```diff
@@ -1798,8 +1869,23 @@ export default class Hyperwallet {
...
+     * Validate the options filter
+     *
+     * @param {Object} inputFilters - The query parameters in the URI
+     * @param {Object} listFilters - Defined list of filters for a business object
+     */
+
+    isValidFilter(inputFilters, listFilters) {
+        return Object.keys(inputFilters).every(elem => listFilters.includes(elem));
+    }
     //--------------------------------------
```
Comment (arao6) [inline src/Hyperwallet.js:1889]: blank line here

File: test/Hyperwallet.spec.js [hunk 2]
```diff
@@ -247,17 +247,20 @@ describe("Hyperwallet", () => {
...
             const callback = () => null;
-            client.listUsers({ test: "value" }, callback);
-
+            client.listUsers({ clientUserId: "test-client-user-id", status: "test-status" }, callback);
             apiClientSpy.should.have.been.calledOnce();
-            apiClientSpy.should.have.been.calledWith("users", { test: "value" });
+            apiClientSpy.should.have.been.calledWith("users", { clientUserId: "test-client-user-id", status: "test-status" });
         });
 
+        /** @test {Hyperwallet#listUsers} */
```
Comment (arao6) [inline test/Hyperwallet.spec.js:255]: pls add blank lines between the it tests

**Review by ramahalingam — APPROVED** [review]
Suggestion: Can we have the error with the allowed filters thrown back to the user in all the cases?

File: src/Hyperwallet.js (hunk 1 above)
Comment (Rrathinasabapath) [reply src/Hyperwallet.js:1889]: done

File: test/Hyperwallet.spec.js (hunk 2 above)
Comment (Rrathinasabapath) [reply test/Hyperwallet.spec.js:255]: done

**Review by arao6 — APPROVED** [review]

**Review by peter-joseph — APPROVED** [review]

**Pull request merged**